etna/models/catboost.py

```diff
@@ -38,8 +38,21 @@ def __init__(
             l2_leaf_reg=l2_leaf_reg,
             **kwargs,
         )
+        self.regressor_columns: Optional[List[str]] = None
         self._categorical = None
 
+    def _select_regressors(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Select data with regressors."""
+        if self.regressor_columns is None:
+            raise ValueError("Something went wrong, regressor_columns is None!")
+
+        if self.regressor_columns:
+            result = df[self.regressor_columns]
```

Comment:
Are we fine with the default `KeyError` from pandas in case of missing regressor columns in `df`? Seems possible when a custom dataset is passed to the forecasts/predict.